### PR TITLE
Add `LightningPoseDataInterface` and `LightningPoseConverter` 

### LINKED ISSUE #628: [New Format]: Lightning Pose
### What format would you like to see added to NeuroConv?

Support writing `PoseEstimationSeries` from [Lightning Pose](https://github.com/danbider/lightning-pose/tree/main) output.

- add helper method `add_pose_estimation_series()` in `neuroconv.tools` which uses [ndx-pose](https://github.com/rly/ndx-pose)
- add LightningPoseInterface in `neuroconv.datainterfaces.behavior`
- extract metadata from csv: scorer, key point names
- extract timestamps from video (should also be able to override)

Example data: https://github.com/danbider/lightning-pose/tree/main/data/mirror-mouse-example

### Does the format have any documentation?

https://github.com/danbider/lightning-pose/blob/main/docs/README.md

### Existing APIs for format

_No response_

### Do you have any example files you are willing to share?

Example data: https://github.com/danbider/lightning-pose/tree/main/data/mirror-mouse-example

### Do you have any interest in helping implement the feature?

Yes.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

### PR DESCRIPTION
**Pull request by weiglszonja**
Add converter for LightningPose, uses `ndx-pose==0.1.0` to add `PoseEstimation` with the `PoseEstimationSeries` from the predictions CSV file. The original video is added as ImageSeries to `nwbfile.acquisition`, while the optional labeled video is added to `nwbfile.processing["behavior"]` both in `external` mode.

Resolve #628 

https://github.com/danbider/lightning-pose/blob/main/docs/directory_structures.md#model-directory-structure

Checklist
- [x] Example data PR: https://gin.g-node.org/CatalystNeuro/behavior_testing_data/pulls/10
- [x] add unittest
- [x] add doc

### CODE REVIEW

File: docs/api/interfaces.behavior.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ FicTrac
 -------
 .. automodule:: neuroconv.datainterfaces.behavior.fictrac.fictracdatainterface
 
+LightningPose
+-------------
+.. automodule:: neuroconv.datainterfaces.behavior.lightningposer.lightningposeconverter
```
Comment (weiglszonja) [inline docs/api/interfaces.behavior.rst:18]:
```suggestion
.. automodule:: neuroconv.datainterfaces.behavior.lightningpose.lightningposeconverter
```

File: docs/api/interfaces.behavior.rst [hunk 2]
```diff
@@ -13,6 +13,10 @@ FicTrac
 -------
 .. automodule:: neuroconv.datainterfaces.behavior.fictrac.fictracdatainterface
 
+LightningPose
```
[the remark this answers is not in the document]
Comment (weiglszonja) [reply docs/api/interfaces.behavior.rst:16]: Yes, thanks for the reminder https://github.com/catalystneuro/neuroconv/pull/633/commits/563d145276730b15b9f176696c78073a8869ce20

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Looks great to me, thanks a bunch @weiglszonja 

The only small detail that I'm OK with since this isn't required for a full publication ready conversion using `main` (unlike a similar consideration given to FicTrac recently) is the lack of `H5DataIO` wrappers around the pose series data, but now that #557 is merged you could technically just add that call to `configure_backend` in the Pinto conversion script to achieve the same effect (this will soon be integrated into the core Interface/Converter classes to be seamless and not require a manual call, ideally prior to next offical release)

**Comment by CodyCBakerPhD** [comment]
Retriggering CI docs...

**Pull request merged**